tests/functionality_test/functionality_testing.py

```diff
@@ -0,0 +1,101 @@
+import time
```

Comment:
@vimalsheoran Please arrange the imports according to the Coding Guidelines.

Reply:
This issue will be fixed and be committed. Thanks for the heads up.